.github/workflows/ci.yml

```diff
@@ -83,19 +83,29 @@ jobs:
           node-version: 16
           cache: 'yarn'
 
+      - name: Cache Turbo
+        uses: actions/cache@v2
+        with:
+          path: "**/.turbo"
+          key: "turbo-${{ github.ref }}-${{ hashFiles('packages/**/src/**') }}-${{ github.sha }}"
+          restore-keys: |
+            turbo-${{ github.ref }}-${{ hashFiles('packages/**/src/**') }}-${{ github.sha }}
+            turbo-${{ github.ref }}-${{ hashFiles('packages/**/src/**') }}-
+            turbo-${{ github.ref }}-
```

Comment:
`turbo` generates `.turbo/` directories in each workspace module. They're `.gitignore`'d but this caches them from this commit, falling back to this ref if that doesn't exist.

This is the key to getting `turbo`'s super powers working in CI.

Reply:
this isn't necessary to cache these are simply logs

Reply:
the only one you want to cache is `node_modules/.cache/turbo`

Reply:
Thanks for the input! Appreciate it very much.